README.md

```diff
@@ -105,6 +105,26 @@ the next 100 are measured for inference throughput.
 The input parameters to the test are used to generate permutations
 of tests with varying configurations.
 
+### The model
+As Neural Network, we use Pytorch's implementation of Resnet50. The script `imagenet/model_saver.py`
+can be used to generate the model for CPU or GPU. 'By navigating to the imagenet folder, it can be invoked with
```

Comment:
remove the lil ' next to the word By

Reply:
It is actually at the end of the paragraph, because I did not want to put too much information before the command were described.

Reply:
Ahhh I see I did not fully read, and good idea